gradle/libs.versions.toml

```diff
@@ -225,4 +225,4 @@ knit = { id = "org.jetbrains.kotlinx.knit", version = "0.5.0" }
 
 # Version '4.3.1.3277' introduced some regressions in CI time (more than 2x slower), so make sure
 # this is no longer the case before upgrading.
-sonarqube = "org.sonarqube:4.2.1.3168"
+sonarqube = "org.sonarqube:4.3.0.3225"
```

Comment:
We used to test `4.3.1.3277` as per the comment above, and I have checked that it's not a typo, and `4.3.0.3225` is below, that's weird.